cloudbuild-tag.yaml

```diff
@@ -11,5 +11,13 @@ steps:
   - gcr.io/$_STAGING_PROJECT_ID/testrunner:sha_$COMMIT_SHA
   - gcr.io/$PROJECT_ID/testrunner:$TAG_NAME
 
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: /bin/bash
+  args:
+  - ./cloudbuild/tag_latest.sh
```

Comment:
I think this should be `/workspace/cloudbuild/tag_latest.sh` instead.

Also, should be possibile to use `tag_latest.sh` directly as an entrypoint.

```yaml
- name: gcr.io/cloud-builders/gcloud
  entrypoint: /workspace/cloudbuild/tag_latest.sh
  args:
  - gcr.io/$PROJECT_ID/testrunner
  - $TAG_NAME
```